tests/functional/example/conftest.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Common fixtures and utils for example tests."""
+
+import os
+import tempfile
+
+import pytest
+
+
+@pytest.fixture(scope="session")
```

Comment:
scope='session' will likely cause issues. It would be preferable that the DB starts from scratch for each test.

Reply:
I was wondering about that, but it seemed to work fine. I'll change it to `function` scope